### ISSUE TITLE
Loading screen HTTPS problems (chromium)

### ISSUE THREAD

**Issue by Brouskat** [Facepunch/garrysmod-issues#3886]
### Details
All my websites running with HTTPS (TLS 1.2 + TLS 1.3) but the loading screen doesn't working at all. It show nothing and doesn't loading the page.

I did some research on my side and I saw that the HTTPS loading screen only takes the protocol TLS 1.0 which is an unsafe protocol and already dating, the recommended minimum is the TLS 1.2.

I will not detail why the TLS 1.0 is currently not recommended, I leave you Google to look for that.

I saw that there is a "chromium" branch on Garry's Mod and to what I've seen, it takes the TLS 1.2 protocol.

When will the loading screen will be upgraded to the chromium branch ? Because of that, forced to activate the TLS 1.0 which puts insecurity on the site.

As well, I see that the chromium branch as been updated about 16 hours ago but, when he will be released ?

It's been a long time since this problem exists and I'm surprised that developers can't do something quickly for this little thing.

Non-chromium branch log : 
`"Mozilla/5.0 (Windows; Valve Source Client) AppleWebKit/535.19 (KHTML, like Gecko) Chrome/18.0.1003.1 Safari/535.19 Awesomium/1.7.5.1 GMod/13"`

Chromium branch log : 
`"Mozilla/5.0 (Windows; Valve Source Client) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.3325.146 Safari/537.36 GMod/13"`

**Comment by xopez** [comment]
Yes it should be. This post is a year old now. This also affects fastdl and addons which loads websites for users such like mapvote with images which are gathered over https. I have already got myself in the beta "x86-64" and not the "chromium" and it works like a charm. But not all users know much about betas or willing to change from stable branch. Don't know how much more developement this needs.

**Comment by viral32111** [comment]
It'll be done when x64 is done & released I suppose, but I'm pretty sure there's no real time frame on that.

I'd like this alot too, so I can finally have full HSTS on my website, but I've just come to accept that it isn't going to happen for a while and thus simply disabled HTTPs on my loading screen & fastdl since they don't *need* to use encrypted connection as there's no sensitive data being exchanged from the game to the web server, but of course HTTPs is more ideal with the modern internet.

I'm pretty sure this also affects in-game DHTML panels and other web-based rendering things, and in those situations a modern and secure connection can be heavily desired.
